src/content/en/tools/lighthouse/audits/geolocation-on-load.md

```diff
@@ -0,0 +1,37 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Page Does Not Automatically Request Geolocation On Page Load" Lighthouse audit.
+
+{# wf_updated_on: 2016-11-30 #}
+{# wf_published_on: 2016-11-30 #}
+
+# Page Does Not Automatically Request Geolocation On Page Load  {: .page-title }
+
+## Why the audit is important {: #why }
+
+Users are mistrustful of or confused by pages that automatically request
+their location on page load. Rather than automatically requesting a
+user's location on page load, tie the request to a user's gesture, such as
+a tapping a "Find Stores Near Me" button. Make sure that the gesture clearly
+and explicitly expresses the need for the user's location.
+
+## How to pass the audit {: #how }
+
+Under **URLs**, Lighthouse reports the line and column numbers where your
+code is requesting the user's location. Remove these calls, and tie the
+requests to user gestures instead. 
+
+See [Ask permission responsibly][ask] for a list of best practices when
+requesting a user's location.
+
+[ask]: /web/fundamentals/native-hardware/user-location/#ask_permission_responsibly
+
+{% include "web/tools/lighthouse/audits/implementation-heading.html" %}
+
+If geolocation permission was already granted to a page before Lighthouse's
+audit, Lighthouse cannot determine if the page requests the user's location
+on page load. Reset the permissions and run Lighthouse again.
```

Comment:
"Reset the permission to the default behavior and run Lighthouse again."

Can we link to a doc on how to nuke permissions? On desktop, it's under the lock icon:

<img width="348" alt="screen shot 2016-12-07 at 10 32 26 am" src="https://cloud.githubusercontent.com/assets/238208/20981200/7886e6ee-bc68-11e6-8bd8-785a66cb3228.png">